src/connection.rs

```diff
@@ -28,7 +45,13 @@ impl COMLibrary {
     ///
     pub fn new() -> WMIResult<Self> {
         let instance = Self::without_security()?;
-        instance.init_security()?;
+
+        match instance.init_security() {
+            Ok(()) => {}
+            // Security was already initialized, this is fine
+            Err(WMIError::HResultError { hres }) if hres == RPC_E_TOO_LATE.0 => {}
```

Comment:
Having to access the `i32` representation of the `HRESULT` with `.0` seems a bit off to me.
Can we change the type of `hres` to `HRESULT`?

Reply:
I agree, but that will be a breaking change so I wanted to avoid that for now.